src/AaveV3ArbNativeUSDCListing_20230621/ADD-NATIVE-USDC-ARB-V3-AIP.md

```diff
@@ -1,9 +1,7 @@
 ---
 title: Add Native USDC To Arbitrum V3 Pool
 author: Marc Zeller (@marczeller), Aave-Chan initiative
-shortDescription: Add Native USDC To Arbitrum V3 Pool
```

Comment:
there's no ui in existence which relies on these field, so better to remove them.